pkgs/applications/misc/synapse/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, intltool, pkgconfig, 
+  glib, libnotify, gtk3, libgee,
+  keybinder3, json_glib, zeitgeist, vala
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "synapse-0.2.99.1";
+
+  src = fetchurl {
+    url = "https://launchpad.net/synapse-project/0.3/0.2.99.1/+download/${name}.tar.xz";
+    sha256 = "846d8a5130580bb47c754bb7f20dc76311e589c00a18b02370a5d78b52409220";
+  };
+
+  buildInputs = [
+    intltool pkgconfig glib 
+    libnotify gtk3 libgee keybinder3
+    json_glib zeitgeist vala
+  ];
+
+  meta = { 
+      longDescription = "Semantic launcher written in Vala that you can use to start applications as
+well as find and access relevant documents and files by making use of the Zeitgeist engine";
+      description = "Semantic launcher to start applications and find relevant files";
+      homepage = https://launchpad.net/synapse-project;
+      license = stdenv.lib.licenses.gpl3;
+      maintainers = with stdenv.lib.maintainers; [
+        "Michal Hruby" "Alberto Aldegheri" "Julien Lavergne" "Rico Tzschichholz"
```

Comment:
Remove these maintainers. Not only this line is broken, but they sure are not the maintainers of this package because its you packaging it.